lib/ts/recipe/passwordless/components/features/signInAndUp/index.tsx

```diff
@@ -38,23 +38,33 @@ import type { PasswordlessSignInUpAction, SignInUpState, SignInUpChildProps, Nor
 import type { RecipeInterface } from "supertokens-web-js/recipe/passwordless";
 import type { User } from "supertokens-web-js/types";
 
-export const useSuccessInAnotherTabChecker = (
-    state: SignInUpState,
-    dispatch: React.Dispatch<PasswordlessSignInUpAction>,
-    userContext: any
-) => {
+export const useRedirectAfterSuccess = (state: SignInUpState, recipeId: string, userContext: any, history: any) => {
```

Comment:
A better name would be `useRedirectAfterSuccessInAnotherTab` - otherwise it's confusing that this is also for the current tab.

Reply:
Outdated. This function has been removed.